contracts/governance/Governor.sol

```diff
@@ -227,6 +220,20 @@ abstract contract Governor is Context, ERC165, EIP712, Nonces, IGovernor, IERC72
         return _proposals[proposalId].proposer;
     }
 
+    /**
+     * @dev Throws if the executor is not the owner.
```

Comment:
```suggestion
     * @dev Reverts if the `msg.sender` is not the executor. In case the executor is not this contract
     * itself, the function reverts if `msg.data` is not whitelisted as a result of an {execute}
     * operation. See {onlyGovernance}.
```